Formula/crumble.rb

```diff
@@ -0,0 +1,19 @@
+class Crumble < Formula
+  desc "Controllable lossy ompression of BAM/CRAM files"
```

Comment:
`ompression` typo?

Reply:
Maybe it was deliberate *lossy compression* !!!

(fixed)